python/invoke_release/tasks.py

```diff
@@ -1198,13 +1259,19 @@ def release(_, verbose=False, no_stash=False):
 
         _pre_commit(__version__, release_version)
 
+        if USE_PULL_REQUEST:
+            branch_name = 'invoke-release-{}'.format(release_version)
```

Comment:
can we have the base as part of the name which would then support hotfix from a branch as well.

for eg. `invoke-release-master-0.1.1` or `invoke-release-0.1.x-0.1.2`

Reply:
Oh, good point, Venkatesh! I'm a big 👍to this suggestion.

Reply:
done, please review the last commit @nickwilliams-eventbrite @venkatesh-eb